packages/samples/headless-react/src/pages/SearchPage.tsx

```diff
@@ -103,11 +103,18 @@ import {
   buildRelevanceInspector,
   StandaloneSearchBox as HeadlessStandaloneSearchBox,
   buildStandaloneSearchBox,
+  SearchAppState,
 } from '@coveo/headless';
 import {bindUrlManager} from '../components/url-manager/url-manager';
 import {setContext} from '../components/context/context';
 import {dateRanges} from '../components/date-facet/date-utils';
 
+declare global {
+  interface Window {
+    HEADLESS_STATE: SearchAppState;
+  }
+}
```

Comment:
I would probably mention somewhere that this is strictly for server-side rendering.